core/src/main/java/com/google/cloud/sql/ConnectorRegistry.java

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
Do we need to add this in this PR?

Reply:
No. I'll move it to the PR that introduces the public API.